src/vs/workbench/contrib/terminal/browser/links/terminalLinkQuickpick.ts

```diff
@@ -20,20 +20,20 @@ export class TerminalLinkQuickpick {
 		const filePicks = links.fileLinks ? await this._generatePicks(links.fileLinks) : undefined;
 		const webPicks = links.webLinks ? await this._generatePicks(links.webLinks) : undefined;
 		const options = {
-			placeHolder: localize('terminal.integrated.openWebLink', "Select the link to open"),
+			placeHolder: localize('terminal.integrated.openDetectedLink', "Select the link to open"),
 			canPickMany: false
 		};
 		const picks: LinkQuickPickItem[] = [];
 		if (webPicks) {
-			picks.push({ type: 'separator', label: localize('terminal.integrated.webLinks', "Web") });
+			picks.push({ type: 'separator', label: localize('terminal.integrated.urlLinks', "Url") });
 			picks.push(...webPicks);
 		}
 		if (filePicks) {
-			picks.push({ type: 'separator', label: localize('terminal.integrated.fileLinks', "File") });
+			picks.push({ type: 'separator', label: localize('terminal.integrated.fileLinks', "Local File") });
```

Comment:
localLinks? We should be careful about changing command name generally but these are new so a bit of change is fine imo

Reply:
we only show local file links, so I think fileLinks is more appropriate here. could go with localFile links though

Reply:
This is fine 👍